iterminator/iterminator.py

```diff
@@ -12,6 +12,24 @@
 import getchs
 
 
+LIGHT_SCHEMES = [
```

Comment:
how do you know which are light?  is there a way to abstract this, for when new schemes are added?